rfcs/0000-universal-asset-notation/0000-universal-asset-notation.md

```diff
@@ -0,0 +1,118 @@
+- `<path>` is a pair of bridge symbol and chain symbol, connected by a single dot `.`.
+- `<asset-symbol>` is a upper-case abbreviation of the asset, e.g. `BTC` for bitcoin, `CKB` for CKByte.
+- `<chain-symbol>` is a lower-case abbreviation of the source chain where the asset resided before came through the path, e.g. `eth` for the Ethereum chain, `ckb` for the CKB chain. Note the source chain may be different from the issuance chain of the asset.
+- `<bridge-symbol>` is a lower-case abbreviation of the bridge through which the asset came, e.g. `fb` for assets moved by Forcebridge.
+
+Examples:
+
+```
+// CKByte on the CKB chain
+CKB.ckb
+
+// CKByte on the Godwoken chain
+CKB.gw
+
+// WBTC crossed from the Ethereum chain to the CKB chain via Forcebridge
+WBTC.ckb|fb.eth
+
+// BNB crossed from the BSC chain to the CKB chain via Forcebridge
+BNB.ckb|fb.bsc
+
+// ETH crossed from the Ethereum chain to the CKB chain via Anotherbridge
+ETH.ckb|ab.eth
+
+// ETH crossed from the Ethereum chain to the Godwoken chain via Forcebridge and Godwoken bridge
+ETH.gw|gb.ckb|fb.eth
+```
+
+## Display Name
+
+Applications can display UAN directly on user interface, or parse UAN for more user-friendly displays. It is up to each application to determine how to display UAN.
+
+One way to display UAN is keeping the basic form with some omissions. For example:
+
+- Omit the chain of residence when the context is clear and there's no ambiguity when user interacts with the application, e.g. `CKB` or `WBTC|fb.eth`.
+- Omit the intermedia paths to emphasis the chain of origin and reduce interference, e.g. use `ETH|fb.eth` for `ETH` crossed from the Ethereum chain and eventually resides on the Godwoken chain, on the UI of a Godwoken dapp.
+
+Another possible format of UAN display could be:
+
+```
+<asset-symbol> " (via " <bridge-name> " from " <chain-symbol-in-upper-case> ")"
+```
+
+Where `<bridge-name>` is the full name of the bridge in the rightmost path of route, and `<chain-symbol-in-upper-case>` is the chain symbol of the chain in the rightmost path of route. The chain of residence is omitted in display name because it should be clear in the context of the dapp with which a user interacts. The intermediate paths are omitted to make display name more readable.
+
+Examples:
+
+```
+// WBTC crossed from the Ethereum chain to the CKB chain via Forcebridge
+// UAN
+WBTC.ckb|fb.eth
+// Display Name
+WBTC (via Forcebridge from ETH)
+
+// BNB crossed from the BSC chain to the CKB chain via Forcebridge
+// UAN
+BNB.ckb|fb.bsc
+// Display Name
+BNB (via Forcebridge from BSC)
+
+// ETH crossed from the Ethereum chain to the CKB chain via Anotherbridge
+// UAN
+ETH.ckb|ab.eth
+// Display Name
+ETH (via Anotherbridge from ETH)
+
+// ETH crossed from the Ethereum chain to the Godwoken chain via Forcebridge and Godwoken bridge
```

Comment:
```suggestion
// ETH crossed from the Ethereum chain to the Godwoken chain via Force Bridge and Godwoken Bridge
```

Reply:
Follow the name style of Force Bridge

## References:
- https://github.com/nervosnetwork/force-bridge/pull/373#discussion_r924029488
- https://github.com/nervosnetwork/rfcs/pull/335/files#r913060381